### PR TITLE
cross publish to 2.13

### PR DESCRIPTION
**Pull request by MasseGuillaume**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -3,9 +3,9 @@ import org.scalajs.core.tools.sem.CheckedBehavior
 
 name               in ThisBuild := "utest"
 organization       in ThisBuild := "com.lihaoyi"
-version            in ThisBuild := "0.4.7-SNAPSHOT"
+version            in ThisBuild := "0.4.8"
```
Comment (japgolly) [inline build.sbt]: Please don't change this line, the next release will be 0.4.7 (latest is 0.4.6)

**Comment by japgolly** [comment]
Hi @MasseGuillaume thanks for the PR. Could you please just revert that one line and then I'll merge.

**Comment by MasseGuillaume** [comment]
Done ;-).

**Comment by japgolly** [comment]
Great, thanks @MasseGuillaume

**Comment by MasseGuillaume** [comment]
Amazing! Can you cut a release?

**Comment by japgolly** [comment]
Absolutely but busy atm, will do by this time tomorrow

**Comment by japgolly** [comment]
@MasseGuillaume 0.4.7 is on its way to Maven Central. :grinning:

**Comment by MasseGuillaume** [comment]
awesome, thanks!

**Comment by lihaoyi** [comment]
@MasseGuillaume if you could update your PRs to my other repos so that their tests pass on the published version of uTest, I'll publish those too.

Probably need to start with lihaoyi/acyclic since many other repos depend on that

**Comment by MasseGuillaume** [comment]
@lihaoyi I will do this first thing tonight (in ~ 10h)

**Comment by japgolly** [comment]
@MasseGuillaume If you're making similar changes, you should also know, there was a small problem with this PR that let's you think everything is ok until the last step in the Sonatype UI where it gets rejected: when replacing `pomExtra` with SBT's DSL you need to make sure that the `homepage` key is specified as that translates to the `<url>` node in the pom. See https://github.com/lihaoyi/utest/commit/24989d1a804c979019d90a3b255f575107bab91e

**Comment by MasseGuillaume** [comment]
@japgolly thanks, noted ;-)

**Pull request merged**